src/extensions/query-translator/component/OverrideCardQueryEditor.tsx

```diff
@@ -132,24 +141,33 @@ export const NeoOverrideCardQueryEditor = ({
                   className='n-ml-2'
                 />
               </td>
-              <td style={{ width: 70 }}>&nbsp; English</td>
-              <td style={{ width: '100px', float: 'right' }}>
+              <td style={{ width: 50, textAlign: 'left', paddingTop: '6px' }}>&nbsp;English&nbsp;</td>
+              <td style={{ float: 'right' }}>
                 {/* Only show translation button if there's something new to translate */}
                 {language == Language.ENGLISH ? (
-                  <Button
-                    fill='outlined'
-                    disabled={prepopulateExtensionName == undefined}
-                    style={{ float: 'right' }}
-                    onClick={() => {
-                      if (prepopulateExtensionName !== undefined) {
-                        triggerTranslation();
-                        setLanguage(Language.CYPHER);
-                        deletePrepopulationReportFunction(reportId);
-                      }
-                    }}
-                  >
-                    Translate
-                  </Button>
+                  <div style={{ display: 'flex', justifyContent: 'flex-end' }}>
```

Comment:
the UX might be confusing here for users, they could wonder
- What's a Q&A?
- Are they global or specific to this report?
- Should I bother creating them?

Perhaps this should all be in the extension config modal. 
How about a second screen opened by the extension modal called 'Tweak Model', where users can add examples etc.?